templates/player.html

```diff
@@ -83,14 +83,14 @@ <h2 class="pull-right">Potential: <span data-bind="text: player.ratings()[player
 <div class="table-responsive">
   <table class="table table-striped table-bordered table-condensed table-clickable-rows" id="player_stats">
     <thead>
-      <tr><th colspan="6"></th><th colspan="3" style="text-align: center" title="Field Goals">FG</th><th colspan="3" style="text-align: center" title="Three-Pointers">3PT</th><th colspan="3" style="text-align: center" title="Free Throws">FT</th><th colspan="3" style="text-align: center" title="Rebounds">Reb</th><th colspan="8"></th></tr>
-      <tr><th>Year</th><th>Team</th><th>Age</th><th title="Games Played">GP</th><th title="Games Started">GS</th><th title="Minutes">Min</th><th title="Made">M</th><th title="Attempted">A</th><th title="Percentage">%</th><th title="Made">M</th><th title="Attempted">A</th><th title="Percentage">%</th><th title="Made">M</th><th title="Attempted">A</th><th title="Percentage">%</th><th title="Offensive">Off</th><th title="Defensive">Def</th><th title="Total">Tot</th><th title="Assists">Ast</th><th title="Turnovers">TO</th><th title="Steals">Stl</th><th title="Blocks">Blk</th><th title="Personal Fouls">PF</th><th title="Points">Pts</th><th title="Player Efficiency Rating">PER</th><th title="Estimated Wins Added">EWA</th></tr>
+      <tr><th colspan="6"></th><th colspan="3" style="text-align: center" title="Field Goals">FG</th><th colspan="3" style="text-align: center" title="Three-Pointers">3PT</th><th colspan="3" style="text-align: center" title="Free Throws">FT</th><th></th><th colspan="3" style="text-align: center" title="Rebounds">Reb</th><th colspan="9"></th></tr>
+      <tr><th>Year</th><th>Team</th><th>Age</th><th title="Games Played">GP</th><th title="Games Started">GS</th><th title="Minutes">Min</th><th title="Made">M</th><th title="Attempted">A</th><th title="Percentage">%</th><th title="Made">M</th><th title="Attempted">A</th><th title="Percentage">%</th><th title="Made">M</th><th title="Attempted">A</th><th title="Percentage">%</th><th title="+/-">+/-</th><th title="Offensive">Off</th><th title="Defensive">Def</th><th title="Total">Tot</th><th title="Assists">Ast</th><th title="Turnovers">TO</th><th title="Steals">Stl</th><th title="Blocks">Blk</th><th title="Blocks Against">BA</th><th title="Personal Fouls">PF</th><th title="Points">Pts</th><th title="Player Efficiency Rating">PER</th><th title="Estimated Wins Added">EWA</th></tr>
```

Comment:
Was going by the box score display at Yahoo Sports and NBA.com, both of them have +/- between field goal stats and rebounding stats, but I'll move them to the end (just before PER / GmSc).